### PR TITLE
Correct version bump

### PR DESCRIPTION
**Pull request by labs-dlugo**
Replaces #10

### CODE REVIEW

File: packages/integration-sdk-core/jest.config.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
 module.exports = {
   ...require('../../jest.config.base'),
-  setupFilesAfterEnv: ['./jest.setup.ts'],
+  //setupFilesAfterEnv: ['./jest.setup.ts'],
```
Comment (sebassdc) [inline packages/integration-sdk-core/jest.config.js]: why is this line commented?\

**Review by sebassdc — CHANGES_REQUESTED** [review]

File: packages/integration-sdk-core/jest.config.js (hunk 1 above)
Comment (labs-dlugo) [reply packages/integration-sdk-core/jest.config.js]: The file doesn't actually exist

Comment (labs-dlugo) [reply packages/integration-sdk-core/jest.config.js]: @sebassdc

Comment (sebassdc) [reply packages/integration-sdk-core/jest.config.js]: @labs-dlugo why dont we delete the line then?

**Review by sebassdc — CHANGES_REQUESTED** [review]

File: packages/integration-sdk-core/jest.config.js (hunk 1 above)
Comment (labs-dlugo) [reply packages/integration-sdk-core/jest.config.js]: ok

**Review by sebassdc — APPROVED** [review]

**Pull request merged**